docs/reviewing.md

```diff
@@ -0,0 +1,15 @@
+# Reviewing policies
+
+The following policies are preferred during the review, but not mandatory:
+
+## Trailing white-spaces
+
+Avoid trailing white-space characters, if possible
+
+## Quotes
+
+Prefer single quotes (`'`) over double quotes (`"`) in python code (`conanfile.py`, `test_package/conanfile.py`)
```

Comment:
This is hot food for debate.
See e.g. this twitter message: https://twitter.com/raymondh/status/1259209765072154624

I would change it to the following:
```
Don't mix single quotes (`'`) and double quotes (`"`) in python code (`conanfile.py`, `test_package/conanfile.py`).
Be consistent.
```

Reply:
yes, I agree.